README.md

```diff
@@ -89,34 +90,33 @@ features = ["stm32f405", "rt"]
 The nightlies should always build and be as stable as the latest release, but
 contain the latest patches and updates.
 
-
 ## Generating Device Crates / Building Locally
 
-* Install `svd2rust`, `svdtools`, and `form`:
```

Comment:
Sorry to be a pain, but could you undo these changes? I guess you have something auto-format markdown files, but this PR doesn't need to change all the bullet point lists!

Reply:
Totally reasonable. I will undo :)